src/bitcoin-cli-res.rc

```diff
@@ -17,13 +17,13 @@ BEGIN
         BLOCK "040904E4" // U.S. English - multilingual (hex)
         BEGIN
             VALUE "CompanyName",        "Bitcoin"
-            VALUE "FileDescription",    "Bitcoin-cli (OSS RPC client for Bitcoin)"
+            VALUE "FileDescription",    "bitcoin-cli (JSON-RPC client for Bitcoin Core)"
```

Comment:
I'm not sure license information belongs in the simple file description (and OSS is not a very common abbreviation in any case)